colibre/scripts/gas_smoothing_lengths.py

```diff
@@ -93,7 +93,6 @@ def make_single_image(filenames, names, h_bounds, number_of_simulations, output_
         ax.axvline(x=h_min, color=line.get_color(), ls="--", lw=0.2)
 
     ax.legend()
-    ax.set_xlim(*h_bounds)
```

Comment:
I removed this so that the lowest limit on h (now set to 1e-4 \times grav soft) is always within the plot's domain.